### PR TITLE
Update Update-AzSynapseWorkspace.md

### PR DESCRIPTION
**Pull request by Max-Meng**
Add command example to set TDE protector for a workspace.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by vidai-msft — CHANGES_REQUESTED** [review]

File: src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md [hunk 1]
```diff
@@ -127,10 +127,19 @@ foreach($uami in $uamis){
...
+
+This commands set the transparent data encryption protector for a workspace.
 
 ### Example 9
 ```powershell
+Update-AzSynapseWorkspace -WorkspaceName ContosoWorkspace -EncryptionKeyName keyName -UseSystemAssignedIdentityInEncryption true
+
+```
+
+This commands updates workspace with user assigned managed identites $uamilist that will cover current identities.
```
[the remark this answers is not in the document]
Comment (Max-Meng) [reply src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md:139]: @vidai-msft , if you read the example carefully, you will find the current document doesn't provide any example to set TDE protector as I proposed. The current Example 9 only demonstrates how to change the identity used by the workspace to access the Azure key vault.  My change is to change the encryption key in the key vault, which is required during the process of encryption key rotation. 
For your reference: 
https://learn.microsoft.com/en-us/azure/synapse-analytics/security/workspaces-encryption

File: src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md [hunk 2]
```diff
@@ -127,10 +127,19 @@ foreach($uami in $uamis){
 Update-AzSynapseWorkspace -Name ContosoWorkspace -UserAssignedIdentityAction Set -UserAssignedIdentityId $uamilist
 ```
 
-This commands updates workspace with user assigned managed identites $uamilist that will cover current identities.
+
+This commands set the transparent data encryption protector for a workspace, and it is using System Assigned Managed Identity to access the Azure Key Vault.
```
Comment (vidai-msft) [inline src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md:131]: This sentence you removed/updated is actually for Example 8. Can you please revert the change?

File: src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (vidai-msft) [reply src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md:139]: This one single command example is not that difficult to construct per the detailed parameters description below. If you want to help our customers to finish the workspace encryption process via automated PowerShell script, can you please provide with complete commands and more context? Thanks for your contribution!

**Review by vidai-msft — CHANGES_REQUESTED** [review]

File: src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md (hunk 2 above)
Comment (Max-Meng) [reply src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md]: > This sentence you removed/updated is actually for Example 8. Can you please revert the change?

sorry, yes, you are right, I have reverted the change.

File: src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Max-Meng) [reply src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md]: > This one single command example is not that difficult to construct per the detailed parameters description below. If you want to help our customers to finish the workspace encryption process via automated PowerShell script, can you please provide with complete commands and more context? Thanks for your contribution!

@vidai-msft , thanks for your suggestion, I have added the complete command to rotate the encryption key, please help to check if anything unclear, thanks

File: src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md [hunk 3]
```diff
@@ -126,11 +126,31 @@ foreach($uami in $uamis){
...
+##Update the workspace and set the temp key as the TDE protector
+Update-AzSynapseWorkspace -WorkspaceName ContosoWorkspace  -EncryptionKeyName TempKey -UseSystemAssignedIdentityInEncryption true
+
+##Note, we need to create a new key version for the original encrytion key of the Azure key vault before moving to next steps. 
+
+##Update the workspace and set the encryption key back after we created a new key version. 
+Update-AzSynapseWorkspace -WorkspaceName ContosoWorkspace -EncryptionKeyName default -UseSystemAssignedIdentityInEncryption true
+
+##Remove the temp key
+Remove-AzSynapseWorkspaceKey -WorkspaceName new-syn-cpf-udp-prd -Name TempKey
```
[the remark this answers is not in the document]
Comment (Max-Meng) [reply src/Synapse/Synapse/help/Update-AzSynapseWorkspace.md:146]: @vidai-msft , sorry, it should be ContosoWorkspace, thanks

**Review by vidai-msft — APPROVED** [review]
